cache-then-network/README.md

```diff
@@ -5,13 +5,13 @@ This recipe illustrates methods to return network requests from either the cache
 ## Difficulty
 Beginner
 
-## Use case
+## Use Case
 One advantage to using service workers is having programmatic control over what you return from cache and what you prefer to load from server.  This example provides you a front-end to experiment with that choice.
 
-## Features and usage
+## Features and Usage
 
 - Configure form to allow or disallow cache and/or network
 - Click "Get new data" to execute requests
 
 ## Category
-Performance
+General Usage
```

Comment:
I'd keep "Performance" here, we might want to add more recipes in that category.